Tests/NetworkRequestTests.swift

```diff
@@ -104,4 +104,61 @@ class NetworkRequestTests: XCTestCase {
         XCTAssertEqual(urlRequest.cachePolicy, cachePolicy, file: file, line: line)
         XCTAssertEqual(urlRequest.timeoutInterval, timeout, file: file, line: line)
     }
+    
+    func test_NetworkRequest_InitEmptyRequest() {
+        let response = EmptyResponse()
+        XCTAssert(true)
+    }
+    
+    struct CachePolicyAndTimeOutRequest: NetworkRequest {
```

Comment:
Please reorganize these structs and tests so that we have the structs at the top and the rest of the tests above the `private` methods above.

Reply:
Done